swagger-confluence-client.yaml

```diff
@@ -44,6 +44,33 @@ paths:
           description: "Returned if there is no space with the given key, or if the calling user does not have permission to view the space."
           schema:
             $ref: '#/definitions/confluenceSpaceError'
+  /content/{pageId}:
+    get:
+      summary: "Gets a confluence page content by id."
+      operationId: "getPage"
```

Comment:
Shouldn't that be getPageById?

Reply:
You're right. That's more specific. I'll get right to it and change it.